File: snippets/insert/pg.go
package main

import (
	"database/sql"
	"fmt"

	_ "github.com/lib/pq"
)

const (
	host     = "localhost"
	port     = 8812
	user     = "admin"
	password = "quest"
	dbname   = "qdb"
)

func main() {
	connStr := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable", host, port, user, password, dbname)
	db, err := sql.Open("postgres", connStr)
	checkErr(err)
	defer db.Close()

	insertSql := `
	  INSERT INTO test
	  VALUES($1, $2);
        `
	_, err = db.Exec(insertSql, "abc", "123")
	checkErr(err)
}

func checkErr(err error) {
	if err != nil {
		panic(err)
	}
}

File: snippets/insert/pg.js
const { Client } = require("pg")

const start = async () => {
  try {
    const client = new Client({
      database: "qdb",
      host: "127.0.0.1",
      password: "quest",
      port: 8812,
      user: "admin",
    })
    await client.connect()

    const res = await client.query("INSERT INTO test VALUES($1, $2);", [
      "abc",
      "123",
    ])

    console.log(res)

    await client.end()
  } catch (e) {
    console.log(e)
  }
}

start()
